src/gateway/run-server.js

```diff
@@ -44,6 +44,8 @@ export const runServer = async (
          */
         .get("/render", asyncHandler(makeRenderHandler(renderFn)));
 
+    app.set("trust proxy", true);
```

Comment:
Was reading up on GAE stuff and this was one of the things I added to try and get HTTPS things working.

https://cloud.google.com/appengine/docs/standard/nodejs/runtime#https_and_forwarding_proxies

We probably don't need it anymore, but I also don't think it's a problem to keep it.

Reply:
If we're keeping this can you add a comment with the link you included above? Thanks!